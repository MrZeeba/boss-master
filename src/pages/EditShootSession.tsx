import React from 'react';
import { StyleSheet, Text } from 'react-native';

export default function EditShootSession() {
  return (
    <Text style={styles.baseText}>
      I am bold
      <Text style={styles.innerText}> and red</Text>
    </Text>
  );
}

const styles = StyleSheet.create({
  baseText: {
    fontWeight: 'bold',
  },
  innerText: {
    color: 'red',
  },
});
