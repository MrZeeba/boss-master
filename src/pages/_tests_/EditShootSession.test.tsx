import React from 'react';
import renderer from 'react-test-renderer';

import EditShootSession from '../EditShootSession';

describe('<Text />', () => {
  it('has 2 child', () => {
    const tree = renderer.create(<EditShootSession />).toJSON();
    expect(tree.children.length).toBe(2);
  });
});

it('renders correctly', () => {
  const tree = renderer.create(<EditShootSession />).toJSON();
  expect(tree).toMatchSnapshot();
});
