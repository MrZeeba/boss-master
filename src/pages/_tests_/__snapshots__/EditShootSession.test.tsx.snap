// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    {
      "fontWeight": "bold",
    }
  }
>
  I am bold
  <Text
    style={
      {
        "color": "red",
      }
    }
  >
     and red
  </Text>
</Text>
`;
